// react-spinners-css/lib/cjs/Ring/index.js
"use strict";
var __assign = (this && this.__assign) || function () {
    __assign = Object.assign || function(t) {
        for (var s, i = 1, n = arguments.length; i < n; i++) {
            s = arguments[i];
            for (var p in s) if (Object.prototype.hasOwnProperty.call(s, p))
                t[p] = s[p];
        }
        return t;
    };
    return __assign.apply(this, arguments);
};
var __spreadArray = (this && this.__spreadArray) || function (to, from) {
    for (var i = 0, il = from.length, j = to.length; i < il; i++, j++)
        to[j] = from[i];
    return to;
};
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
var jsx_runtime_1 = require("react/jsx-runtime");
var classnames_1 = __importDefault(require("classnames"));
var style_module_css_1 = __importDefault(require("./style.module.css"));
function Ring(_a) {
    var _b = _a.color, color = _b === void 0 ? '#7f58af' : _b, _c = _a.size, size = _c === void 0 ? 80 : _c, className = _a.className, style = _a.style;
    var circles = __spreadArray([], Array(4)).map(function (_, index) {
        return (jsx_runtime_1.jsx("div", { style: {
                borderColor: color + " transparent transparent transparent",
                width: size * 0.8,
                height: size * 0.8,
                margin: size * 0.1,
                borderWidth: size * 0.1,
            } }, index));
    });
    return (jsx_runtime_1.jsx("div", __assign({ className: classnames_1.default(style_module_css_1.default['lds-ring'], className), style: __assign({ width: size, height: size }, style) }, { children: circles }), void 0));
}
exports.default = Ring;
